app/components/list/index.tsx

```diff
@@ -34,6 +34,8 @@ export const List = ({
     newButtonContent: string;
     buttonProps?: any;
   };
+  link?: ListItemProps["link"];
```

Comment:
Can you please add some comments on the type definition so we can see them when we hover the prop in vscode. Same for onRowClick.